config/params.ini

```diff
@@ -4,23 +4,23 @@ name=Botty
 monitor=0
 custom_message_hook=
 logg_lvl=debug
-max_game_length_s=380
+max_game_length_s=100
```

Comment:
Please leave params in param.ini to default unless it is necessary and makes sense for all usecases.

Reply:
changed back to default